src/quo/animated.cljs

```diff
@@ -140,8 +145,10 @@
            (clj->js config)))
 
 (defn spring [clock-value opts config]
-  (.spring ^js animated clock-value
-           (clj->js opts) (clj->js config)))
+  (.spring ^js animated
+           clock-value
+           (clj->js opts)
+           (clj->js config)))
```

Comment:
The formatter's logic here is 
- if all the arguments are on the same line with the fn and the width is within the limit, then no formatting needed. 
- If the arguments not on the same line with the fn, then one argument per line.

So this can be controlled while editing the code